exercises/simple-linked-list/instructions.md

```diff
@@ -0,0 +1,25 @@
+# Instructions
+
+Your task is to create a prototype for a playlist feature for a music player application.
+
+For the prototype, each song will simply be represented by a number.
+The import/export functionality will use arrays.
```

Comment:
My only remaining concern is with the word `arrays`, as not all languages have arrays or might prefer implementing them as lists/vectors or something else. Could we use `collections` or something like that?

Reply:
Oh, yes! Great point.

Reply:
Ok, I think that is better. Might be a bit clunky, but maybe someone upstream will have some good suggestions for that.